src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -77,6 +77,33 @@ public static ReadOnlyAddressBook getSampleAddressBook() {
         return sampleAb;
     }
 
+    /**
+     * Returns a family condition set containing the list of strings given.
+     */
+    public static Set<FamilyCondition> getFamilyConditionSet(String... strings) {
+        return Arrays.stream(strings)
+            .map(FamilyCondition::new)
+            .collect(Collectors.toSet());
+    }
+
+    /**
+     * Returns a food preference set containing the list of strings given.
+     */
+    public static Set<FoodPreference> getFoodPreferenceSet(String... strings) {
+        return Arrays.stream(strings)
+            .map(FoodPreference::new)
+            .collect(Collectors.toSet());
+    }
+
+    /**
+     * Returns a hobby set containing the list of strings given.
+     */
+    public static Set<Hobby> getHobbySet(String... strings) {
+        return Arrays.stream(strings)
+            .map(Hobby::new)
+            .collect(Collectors.toSet());
+    }
+
```

Comment:
could this be generalised into a single method to reduce code duplication?

Reply:
I think its possible, but perhaps if we could consider changing it in the next iteration? As I tried to generalised them, there are quite alot of changes needed to be made including their respective classes and constructors due to casting etc. 
What do you think?

Reply:
sure sound good